
        public class MyClass2 {
            public static void TestTargetFunc() { throw new Exception("ErrorCode:9999"); }
        }

        [TestCase]
        public void TestThrowsConst()
        {
            // Test OK.
            Assert.Throws(
                Is.TypeOf&lt;Exception&gt;()
                .And.Message.EqualTo("ErrorCode:9999"),
                () =&gt; MyClass2.TestTargetFunc()
            );
        }

